src/languages/dns.js

```diff
@@ -0,0 +1,33 @@
+/*
+Language: DNS Zone file
+Author: Tim Schumacher <tim@datenknoten.me>
+Category: common, config
```

Comment:
I copied this from nginx.js, since both have more or the same featureset. What should I do?